src/common/exif.cc

```diff
@@ -572,6 +572,11 @@ static bool dt_exif_read_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     {
       dt_strlcpy_to_utf8(img->exif_lens, sizeof(img->exif_lens), pos, exifData);
     }
+    else if((pos = exifData.findKey(Exiv2::ExifKey("Exif.PentaxDng.LensType"))) != exifData.end()
```

Comment:
This will break on older exiv2 versions. It probably needs a version check like we do with Exif.CanonFi.FocusDistanceUpper.